File: swagger/models/get_sales_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetSalesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		sales
		Contains the Sale objects, each of which describes the sale and
		payment for a purchase event.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Sales': 'sales',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'sales': 'Sales',
		}

	pagination_response = None
	sales = None

File: swagger/Models/GetSalesResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetSalesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse
	 */
	public $paginationResponse;
	/**
	 * Contains the Sale objects, each of which describes the sale and
	 * payment for a purchase event.
	 * 
	 * @var Sale[]
	 */
	public $sales;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class],
			'Sales' => ['sales', Sale::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => 'PaginationResponse',
			'sales' => 'Sales',
			];
	}
}
